python/vowpalwabbit/pyvw.py

```diff
@@ -1158,6 +1160,72 @@ def __str__(self):
         )
 
 
+class CCBLabelType(enum.Enum):
```

Comment:
This is used by both `Slates` & `CCB`. Perhaps we can name it as just `LabelType`?